autoprecompiles/src/lib.rs

```diff
@@ -249,12 +234,8 @@ pub fn generate_precompile<T: FieldElement>(
                 let local_identities = machine
                     .constraints
                     .iter()
-                    .map(|expr| {
-                        //let mut expr = expr.expr.clone();
-                        //powdr::append_suffix_mut(&mut expr, &ssa_counter.to_string());
-                        SymbolicConstraint {
-                            expr: expr.expr.clone(),
-                        }
+                    .map(|expr| SymbolicConstraint {
+                        expr: expr.expr.clone(),
```

Comment:
This still needs to have its references to cols replaced by the new names from above, right?